models/users.js

```diff
@@ -1,22 +1,20 @@
-// Dependencies
-// =============================================================
 
-var Sequelize = require("sequelize");
-var db = require("../config/connection.js");
+module.exports = function(sequelize, DataTypes) {
 
-// Creates a "Users" model that matches up with DB
-var Users = db.define("users", {
-    username: Sequelize.STRING,
-    password: Sequelize.STRING,
-    name: Sequelize.STRING,
-    location: Sequelize.STRING,
-    favorites: Sequelize.STRING,
-    preferences: Sequelize.STRING,
-    has_preferences: Sequelize.BOOLEAN
-});
+    // Creates a "Users" model that matches up with DB
+    var Users = sequelize.define("Users", {
+        userId: DataTypes.STRING,
```

Comment:
Why did you add `userId`? I think sequelize creates the field `id` for us already.